README.md

```diff
@@ -13,8 +13,9 @@
 </div>
 
 ## Contribute
+
 Create branch with your product killedbygoogle exe: `name-product-killed` . 
-Never use the master branch to create PR. read this link: https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github 
+Never use the master branch to create PR. read this link: https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github.
```

Comment:
Let's use markdown to format this.

```
Never use the `master` branch to create a pull request. If you need help, read [How to Create a Pull Request on GitHub](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github).
```

Reply:
Oh, I am going to correct than :D  thank you